src/jsxc.lib.gui.js

```diff
@@ -2360,6 +2360,7 @@ jsxc.gui.window = {
     */
    open: function(bid) {
       var win = jsxc.gui.window.init(bid);
+      jsxc.xmpp.mam.getHistory(bid);
```

Comment:
Please use an event handler for this. I think the appropriate event would be `init.window.jsxc`.